README.md

```diff
@@ -16,10 +16,12 @@ Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in P
     -   `TIFF`
     -   `LIF`
     -   Any additional format supported by [imageio](https://github.com/imageio/imageio)
+        (`PNG`, `GIF`, etc.)
 -   Supports writing metadata and imaging data for:
     -   `OME-TIFF`
--   Supports reading from and writing to any
-    [fsspec](https://github.com/intake/filesystem_spec) supported file system:
+-   Supports reading and writing to
+    [fsspec](https://github.com/intake/filesystem_spec) supported file systems
+    wherever possible:
```

Comment:
for clarity How does it support these mechanisms, so someone who's unfamiliar with fsspec can understand what's involved. [added] Alternatively put a reference link to the remote image reading section.